app/models/bestuursorgaan-classificatie-code.js

```diff
@@ -1,14 +1,19 @@
 import Model, { attr, hasMany } from '@ember-data/model';
-import { collect } from '@ember/object/computed';
 
-export default Model.extend({
+export default class BestuursorgaanClassificatieCodeModel extends Model {
+  @attr label;
+  @attr scopeNote;
+  @attr uri;
+  @hasMany('bestuursfunctie-code', { inverse: 'standaardTypeVan' }) standaardType;
+  @hasMany('bestuursorgaan', { inverse: null }) isClassificatieVan;
+
   // A string representation of this model, based on its attributes.
   // This is what mu-cl-resources uses to search on, and how the model will be presented while editing relationships.
-  stringRep: collect.apply(this,['id', 'label', 'scopeNote']),
-
-  label: attr(),
-  scopeNote: attr(),
-  standaardType: hasMany('bestuursfunctie-code', { inverse: 'standaardTypeVan' }),
-  isClassificatieVan: hasMany('bestuursorgaan', { inverse: null}),
-  uri: attr()
-});
+  get stringRep() {
```

Comment:
I couldn't find any references in the codebase to this property. Is this still needed?

Reply:
I believe this used to be used in an auto generated admin interface, where this probably originated. should be safe to remove now.